modules/runtime/nuxeo-runtime/src/main/java/org/nuxeo/runtime/model/DefaultComponent.java

```diff
@@ -63,6 +65,10 @@ public void deactivate(ComponentContext context) {
         setModifiedNow();
     }
 
+    protected void addRuntimeMessage(Level level, String message) {
```

Comment:
I'm good with using `java.util.logging.Level` in this API, as it's a JDK class.

Reply:
My issue is that we are mixing 2 logging API (even if it's just for a LogLevel...) in all classes using this API - I don't find it clear.

Reply:
Introduced a specific level for this, to still avoid duplicating error-related and warning-related api, and hopefully keeping clear semantics